packages/office-ui-fabric-react/src/utilities/grid/GridCell.types.ts

```diff
@@ -71,4 +74,19 @@ export interface IGridCellProps<T> {
  * tooltip if is this item is normal
  */
   label?: string;
+
+  /**
+  * Method to provide the classnames to style a button.
+  * The default value for this prop is the getClassnames func
+  * defined in BaseButton.classnames.
+  */
+  getClassNames?: (theme: ITheme,
```

Comment:
These class props are all over the map. We have a className, we have cellDisabledStyle (which is actually an array of classnames), and now getClassNames, which is the previous model for providing classes.

I am honestly confused by GridCell in general, and confused as to how it got here. It seems totally unneeded. It's a wrapper around a CommandButton...which does just about nothing extra. It renders a `CommandButton`.

Reply:
I agree. I wanted to remove cellDisabledStyle and cellIsSelectedStyle... but that would be a breaking change. Maybe I can deprecate them? I am using getClassNames because I needed a way to pass the previous classNames to merge them together (to avoid the codepen repro I sent you). 

I think GridCell was created to have a common component for gridNavigation but at this point I see it also as a wrapper, not providing anything extra.